ci-operator/step-registry/mirror-images/tag-images/mirror-images-tag-images-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+function mirror_tag_images () {
+    tag_images_list="/var/run/vault/mirror-registry/tag_images_list"
```

Comment:
I think put image list in vault is not a good choice, it's out of version control, maybe we can just put it in this file, or another repo, wdyt?

Reply:
Yes, but for those images: https://gitlab.cee.redhat.com/aosqe/flexy-templates/-/blob/master/functionality-testing/aos-4_11/hosts/image-list-tag.txt. It is used as testing images, no ship. So, we don't need to care about their versions. And, it's very convenient for the user to update them if we put them in Vault.

Reply:
Personally vault is secret place, mainly used to save some credentials info, the make me feel like vault is abused here.

Reply:
I understand, but it's really the best way to manage those tag images.

Reply:
One more thing to note, not everyone in QE has the permission to update Vault.

Reply:
Yes, removed it.